src/org/jetbrains/kotlin/test/helper/utils.kt

```diff
@@ -74,7 +74,7 @@ fun glob(searchPattern: String, run: (Path) -> Unit) {
     )
 }
 
-private val supportedExtensions = listOf("kt", "kts", "args", "nkt")
+private val supportedExtensions = listOf("kt", "kts", "args", "can-freeze-ide")
```

Comment:
Is this already reflected in the repo? Maybe we should keep both entries here for a while? Or maybe it's not that important...

Reply:
> Is this already reflected in the repo?

[It should be reflected now.](https://github.com/JetBrains/kotlin/compare/382193f44041...937e48241a5b)

> Maybe we should keep both entries here for a while? Or maybe it's not that important...

IDK, ~5-10 test data files don't seem worth introducing tech debt over.